specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/consumption.json

```diff
@@ -58,7 +58,7 @@
         "parameters": [
           {
             "name": "scope",
```

Comment:
I thought we were going to get rid of the `scope` parameters and explicitly specify the paths. Use of a `scope` parameter makes for a very hard-to-use SDK.

Reply:
Oh, did you open a separate PR to do this?

Reply:
@fearthecowboy 2359 is for marketplace. This one is for pricesheet and budgets.